packages/server/lib/config_options.ts

```diff
@@ -151,8 +151,7 @@ export const options = [
     isInternal: true,
   }, {
     name: 'nodeVersion',
-    defaultValue: 'default',
```

Comment:
The default value is removed so that the depreciation warning will only fire when the 'nodeVersion' config is present.